internal/rbd/nodeserver.go

```diff
@@ -188,6 +201,11 @@ func populateRbdVol(
 	// get rbd image name from the volume journal
 	// for static volumes, the image name is actually the volume ID itself
 	if isStaticVol {
+		if req.GetVolumeContext()["migration"] == "true" {
```

Comment:
why we are adding both migration and staticVolume to the attributes? this is just to treat rbd images as static one even though its migrated?

Reply:
yep, an extra marker to make its a migrated static volume.